source/SkiaSharp.Views.Forms/SkiaSharp.Views.Forms.Tizen/SKImageSourceHandler.cs

```diff
@@ -17,6 +18,31 @@ public sealed class SKImageSourceHandler : IImageSourceHandler
 	{
 		private StreamImageSourceHandler handler = new StreamImageSourceHandler();
 
+		public Task<bool> LoadImageAsync(ElmImage image, ImageSource imageSource, CancellationToken cancelationToken = default(CancellationToken))
+		{
+			ImageSource newSource = null;
+
+			switch (imageSource)
+			{
+				case SKImageImageSource imageImageSource:
+					newSource = ImageSource.FromStream(() => ToStream(imageImageSource.Image));
+					break;
+				case SKBitmapImageSource bitmapImageSource:
+					newSource = ImageSource.FromStream(() => ToStream(SKImage.FromBitmap(bitmapImageSource.Bitmap)));
+					break;
+				case SKPixmapImageSource pixmapImageSource:
+					newSource = ImageSource.FromStream(() => ToStream(SKImage.FromPixels(pixmapImageSource.Pixmap)));
+					break;
+				case SKPictureImageSource pictureImageSource:
+					newSource = ImageSource.FromStream(() => ToStream(SKImage.FromPicture(pictureImageSource.Picture, pictureImageSource.Dimensions)));
+					break;
+			}
+
+			// TODO: until we update to 4.4
+			dynamic placeholder = image;
```

Comment:
Not a fan of this, but this allows me to not use the latest version of a few days ago - and still target 4.2.

Reply:
You do not update the application to support more applications that still use version 4.2 and higher, am I right?

Reply:
I think I understand, and yes. This will fix the issue, and still support 4.2+. No need to update the Forms you are on.